src/main/java/org/jsoup/select/NodeFilter.java

```diff
@@ -0,0 +1,82 @@
+package org.jsoup.select;
+
+import org.jsoup.nodes.Node;
+
+/**
+ * Node filter interface. Provide an implementing class to {@link NodeTraversor}
```

Comment:
You seem to be using a maximum line length of 80. That feels a bit anachronistic. It's also not consistent with the rest of the code base. The maximum line length there seems to be 120.